Tasks/VsTest/helpers.ts

```diff
@@ -0,0 +1,121 @@
+import * as fs from 'fs';
+
+import tl = require('vsts-task-lib/task');
+import tr = require('vsts-task-lib/toolrunner');
+import path = require('path');
+import Q = require('q');
+import models = require('./models')
+
+var os = require('os');
+var uuid = require('node-uuid');
+var xml2js = require('xml2js');
+var parser = new xml2js.Parser();
+var builder = new xml2js.Builder();
+
+export class Constants {
+    public static vsTestVersionString = 'version';
+    public static vsTestLocationString = 'location';
+}
+
+export class Helper{
+    public static addToProcessEnvVars(envVars: { [key: string]: string; }, name: string, value: string) {
+        if (!this.isNullEmptyOrUndefined(value)) {
+            envVars[name] = value;
+        }
+    }
+
+    public static setEnvironmentVariableToString(envVars: { [key: string]: string; }, name: string, value: any) {
+        if (!this.isNullEmptyOrUndefined(value)) {
+            envVars[name] = value.toString();
+        }
+    }
+
+    public static isNullEmptyOrUndefined(obj) {
+        return obj === null || obj === '' || obj === undefined;
+    }
+
+    public static isNullOrUndefined(obj) {
```

Comment:
I checked the use cases. aren't they all supposed to use `isNullOrWhitespace` and it's very well written previously. no reinvention